### PR TITLE
chore(Skeleton): deprecate enums

### PR DESCRIPTION
**Pull request by rivka-ungar**
https://monday.monday.com/boards/3532714909/views/80492480/pulses/7021051396

### CODE REVIEW

File: packages/core/src/components/Skeleton/Skelton.types.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export type SkeletonType = "circle" | "rectangle" | "text";
+
+export type TextSkeletonSize = "h1" | "h2" | "h3" | "h4" | "h5" | "h6" | "p" | "small";
```
Comment (talkor) [inline packages/core/src/components/Skeleton/Skelton.types.ts:3]: What about the 'custom'?

Comment (rivka-ungar) [reply packages/core/src/components/Skeleton/Skelton.types.ts:3]: The custom is included in SkeletonSizeType, it should also be added here?

Comment (talkor) [reply packages/core/src/components/Skeleton/Skelton.types.ts:3]: TextSkeletonSize shouldn't be exported, no?

**Review by talkor — APPROVED** [review]

File: packages/core/src/components/Skeleton/Skelton.types.ts (hunk 1 above)
Comment (rivka-ungar) [reply packages/core/src/components/Skeleton/Skelton.types.ts]: Your right, removed the export

**Pull request merged**